web/packages/teleterm/src/ui/TopBar/NavigationMenu/NavigationItem.tsx

```diff
@@ -26,28 +28,18 @@ export function NavigationItem({ item, closeMenu }: NavigationItemProps) {
   };
 
   return (
-    <ListItem p={2} pl={0} gap={2} onClick={handleClick}>
-      <IconBox p={2}>{item.Icon}</IconBox>
+    <StyledListItem onClick={handleClick}>
```

Comment:
```suggestion
    <StyledListItem as="button" type="button" onClick={handleClick}>
```

😏

Reply:
Oops, I wrapped these `<li>` into `<menu>` element, but it doesn't make them interactive 🤦